README.md

```diff
@@ -5,368 +5,17 @@ https://www.simplereport.gov/
 [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=CDCgov_prime-data-input-client&metric=alert_status)](https://sonarcloud.io/dashboard?id=CDCgov_prime-data-input-client) [![Sonar coverage](https://shields.io/sonar/coverage/CDCgov_prime-data-input-client?server=https://sonarcloud.io)](https://sonarcloud.io/dashboard?id=CDCgov_prime-data-input-client)
 
 ## Table of Contents
+  - [Overview](#overview)
+  - [Getting Started](#getting-started)
+  - [Contributing](#contributing)
 
-  - [Developing locally](#developing-locally)
-  - [Running outside of docker](#running-outside-of-docker)
-  - [Support admin](#support-admin)
-  - [Roles](#roles)
-  - [Database cleaning](#database-cleaning)
-  - [Rollbacks](#rollbacks)
-  - [API testing](#api-testing)
-  - [Tests](#tests)
-  - [E2E tests](#e2e-tests)
-  - [Twilio](#twilio)
-  - [MailHog](#mailhog)
-  - [Frontend](#frontend)
-    - [Linters](#linters)
-    - [Storybook and Chromatic](#storybook-and-chromatic)
-  - [PR Conventions](#pr-conventions)
-  - [Cloud Environments](#cloud-environments)
-  - [Cloud Architecture](#cloud-architecture)
-  - [Deploy](#deploy)
-    - [Revert to a Previous Release](#revert-to-a-previous-release)
-    - [Deploy With Action](#deploy-with-action)
+## Overview
+SimpleReport, as part of PRIME (Pandemic-Ready Interoperability Modernization Effort), aims to improve data quality, completeness and timeliness of public health case reporting for COVID-19, the flu and future infectious diseases.
 
-## Developing locally
+SimpleReport does this by creating a real-time avenue for providers who do not have traditional healthcare IT systems to send high quality, timely, structured data to public health. This ensures that public health has a clear(er) and more timely, complete picture of COVID positivity and can take appropriate steps to intervene and prevent further community spread.
 
-The Simple Report application stack consists of:
-- PostgreSQL database
-- Java Spring Boot backend
-- React frontend
+## Getting Started
```

Comment:
⚠️ Please feel free to edit this wiki page with links to other wiki pages that would be helpful for a new dev to SR!